compiler/luci/pass/src/ConvertNCHWToNHWCPass.test.cpp

```diff
@@ -530,6 +530,27 @@ class MaximumGraph final : public SimpleGraph
   luci::CircleConst *limit = nullptr;
 };
 
+class MaximumNonConstGraph final : public SimpleGraph
+{
+protected:
+  loco::Node *insertGraphBody(loco::Node *input) override
+  {
+    max = g.nodes()->create<luci::CircleMaximum>();
+    max->dtype(loco::DataType::FLOAT32);
+    max->shape({1, 16, 4, 4});
+
+    max->x(input);
+    max->y(input);
```

Comment:
How about create two input graph for this?

Reply:
That could be another test. The graph with a single input can test what this PR intends to implement.